### ISSUE TITLE
Regarding Training

### ISSUE THREAD

**Issue by SanthoshRajendiran** [suruoxi/BANet#1]
Hey. Good to see this repository. We would like to use the same BANet with Keras and Tensorflow. Should we need a separate dataset with boundary information alone? And as per the official implementation, all 3 losses, should be used together. Could you help us understand how to pass two different segmentation masks as labels for training.
